### PR TITLE
Database: update to EPSG v10.057

### PR DESCRIPTION
**Pull request by rouault**
CC @dalia2 : this update adds EPSG transformations for the 3 Icegeoid grids, consequently I've removed the PROJ custom transformations and just modified the grid_aliases to point the .gri files referenced by EPSG to PROJ-data's .tif files. I hope this is correct, but I let you confirm.

### CODE REVIEW

File: data/sql/extent.sql [hunk 1]
```diff
@@ -1827,9 +1827,9 @@ INSERT INTO "extent" VALUES('EPSG','2847','Sweden - 2.5 gon W','Sweden - commune
...
 INSERT INTO "extent" VALUES('EPSG','2850','Sweden - 5 gon E','Sweden - east of approximately 21°26''E. See information source for map.',65.24,68.58,21.34,24.17,0);
-INSERT INTO "extent" VALUES('EPSG','2851','Iceland - onshore west of 24°W','Iceland - onshore west of 24°W.',64.71,65.86,-24.66,-24.0,0);
-INSERT INTO "extent" VALUES('EPSG','2852','Iceland - onshore 24°W to 18°W','Iceland - onshore between 24°W and 18°W.',63.34,66.52,-24.0,-18.0,0);
-INSERT INTO "extent" VALUES('EPSG','2853','Iceland - onshore east of 18°W','Iceland - onshore east of 18°W.',63.45,66.59,-18.0,-13.38,0);
+INSERT INTO "extent" VALUES('EPSG','2851','Iceland - mainland west of 24°W','Iceland - mainland west of 24°W.',64.71,65.85,-24.66,-24.0,0);
+INSERT INTO "extent" VALUES('EPSG','2852','Iceland - mainland 24°W to 18°W','Iceland - mainland between 24°W and 18°W.',63.34,66.52,-24.0,-18.0,0);
+INSERT INTO "extent" VALUES('EPSG','2853','Iceland - mainland east of 18°W','Iceland - mainland east of 18°W.',-63.45,66.59,-18.0,-13.38,0);
```
Comment (rhuijben) [inline data/sql/extent.sql]: This sign change looks suspicious...

Comment (rouault) [reply data/sql/extent.sql]: good catch. I've notified IOGP about that, and added a manual commit to fix it in our database (I expect this trivial issue to be fixed in the next EPSG update)

**Comment by RogerLott** [comment]
The bbox values are automatically generated from the polygon. The minor edits to the polygon did not impact the bbox values and should not have changed. We are not 100% certain how this glitch happened and it was certainly unintended, but we suspect it was inadvertant human intervention. A release to correct the issue is about to be triggered. Thanks for bringing the issue to our attention.
Roger Lott
For IOGP

**Comment by rouault** [comment]
@dalia2  Gentle ping to know if we've your green light to proceed

**Comment by dalia2** [comment]
Hello, I will give you an answer in one day.
________________________________

**Comment by dalia2** [comment]
Hi, I looked through. Everything looks fine.

**Comment by RogerLott** [comment]
An error in the sign of the bounding box south latitude for Extent 2853 which was introduced in EPSG v10.057 on 2022-03-19 was corrected through the EPSG v10.058 release of 2022-03-23. In the v10.057 release some small changes were made to the Extent's polygon but none of these impacted the bounding box which now is restored to its values from EPSG v10.055 and previous releases.

 

I understand that this error may have been manually corrected in the Proj copy of EPSG v10.057. Please check.

 

Roger Lott

For IOGP Geodesy Subcommittee

**Comment by rouault** [comment]
> I understand that this error may have been manually corrected in the Proj copy of EPSG v10.057

yes, it was corrected, and since then we've upgraded up to v10.076

**Pull request merged**